lib/src/blockdev.rs

```diff
@@ -83,8 +84,17 @@ pub(crate) struct LoopbackDevice {
 impl LoopbackDevice {
     // Create a new loopback block device targeting the provided file path.
     pub(crate) fn new(path: &Path) -> Result<Self> {
+        let direct_io = match env::var("BOOTC_DIRECT_IO") {
+            Ok(val) => {
+                if val == "on" { "on" } else { "off" }
+            },
+            Err(_e) => {
+                "off"
+            },
+        };
```

Comment:
Hmm, it seems simpler to pass through the variable if present and non-empty directly to the command...something like:

```
        let direct_io_arg = env::var_os("BOOTC_INSTALL_DIRECT_IO").map(|v| format!("--direct-io={v:?}");
...
     .args(["--show", "-P", "--find"])
     .arg(direct_io_arg)
```
or so?  (not tested)